### ISSUE TITLE
Add Python 3.4 support

### ISSUE THREAD

**Issue by tofgarion** [kdeldycke/maildir-deduplicate#30]
Hi,

I have tried to use mdedup both via pip install or by cloning the repo and I have an error with `__init__.py`:

```
 File "SOME_LOCAL_PATH/maildir_deduplicate/__init__.py", line 94, in <module>
    from deduplicate import Deduplicate
ImportError: No module named 'deduplicate'
```

This can be fixed by replacing

`from deduplicate import Deduplicate`

by 

`from maildir_deduplicate.deduplicate import Deduplicate`

I am currently using Python 3.4 on Debian.

Best regards,

Christophe

**Comment by kdeldycke** [comment]
Thanks for the bug report. Unfortunately the code of this project [only run on Python 2.7](https://github.com/kdeldycke/maildir-deduplicate/blob/v1.0.1/setup.py#L77-L79).

I'll be happy to merge any pull-request adding support for broader Python versions.

In the mean time, I'm renaming this ticket to let it aim at Python 3.4 support.

**Comment by kdeldycke** [comment]
Forget what I said. I'm just not awake yet. Your fix is valid. I'm going to address it once I'll regain my focus.

**Comment by kdeldycke** [comment]
v1.1.0 has been released with support of Python 3.4 and 3.5: https://pypi.python.org/pypi/maildir-deduplicate/1.1.0
